python/servo/mutation/test.py

```diff
@@ -0,0 +1,35 @@
+import fileinput
+import re
+import subprocess
+import sys
+
+def mutate_line(file_name, line_number):
+    lines = open(file_name, 'r').readlines()
+    lines[line_number - 1] = re.sub(r'\s&&\s', ' || ', lines[line_number - 1])
+    out = open(file_name, 'w')
+    out.writelines(lines)
+    out.close()
+
+def mutation_test(file_name):
+    lineNumbers = []
+    for line in fileinput.input(file_name):
+        if re.search(r'\s&&\s', line):
+            lineNumbers.append(fileinput.lineno())
+
+    for lineToMutate in lineNumbers:
+        print "Mutating {0} at line {1}".format(file_name, lineToMutate)
```

Comment:
This is testing every mutation exhaustively, but only one mutation at a time. Can we randomize this? Test more than one mutation at a time?

Reply:
Yes, we could as it would reduce the running time of the test significantly. But for now, we have implemented one strategy to mutate, which replaces && to ||. There could be 20 occurrences of this in one file. Consider, only one of the occurrences is where the test coverage is not present. It might take 20+ test runs to find that particular failure. This could be overseen as an intermittent issue. As of now, we have not thought about how the final test report should look like. If we need any report after the end of each test run, then this randomization would make it difficult to produce a report.

Reply:
If we run more than one mutation at a time and trigger the test suite. How do we classify the test failures based on the mutation?